tests/test_database.py

```diff
@@ -527,6 +552,30 @@ def test_insert_many_personal_check_ins(database, many_personal_check_ins):
 			assert result['category'] == category
 			assert result['patient_id'] == patient_id
 
+def test_insert_many_medications(database, many_medications):
+
+	patient_id = many_medications[1]["patient_id"]
+	device_id = many_medications[1]["device_id"]
+	scheduled_time = many_medications[1]["scheduled_time"]
+	response = many_medications[1]["response"]
+	
+	num_test = len(find_all_test(database))
```

Comment:
use the find_all_medication() function. And rename the "test" variables to be medication variables